src/pip/_vendor/packaging/__about__.py

```diff
@@ -18,7 +17,7 @@
 __summary__ = "Core utilities for Python packages"
 __uri__ = "https://github.com/pypa/packaging"
 
-__version__ = "20.9"
+__version__ = "20.10.dev0"
```

Comment:
We do not vendor in-development branches. It'd be better to ask upstream packaging to make a release first.